src/snyk/common/experiment/amplitudeExperiment.ts

```diff
@@ -0,0 +1,7 @@
+// import { Experiment } from '@amplitude/experiment-node-server';
```

Comment:
Why the code comments? Could you remove them?

Reply:
Oops, file from another branch committed, reverting it :)